src/app/services/register/register.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RegisterService {
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  sendRegistrationInfo (username: string, password: string, confirmPsw: string): Observable<any> {
+    return this.http
+    .post(environment.hostName + '/register',
+    {
+      'username': username,
+      'password': password,
+      'confirmPsw': confirmPsw
+    },
+    {observe: 'response'});
```

Comment:
why is this object needed with response?

Reply:
corrected